02-express-tutorial/routes/people.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express")
+const router = express.Router()
+const { addPerson, getPeople, getPerson, updatePerson, deletePerson } = require("../controllers/people")
+
+router.route('/').get(getPeople).post(addPerson)
+router.route('/:id').get(getPerson).put(updatePerson).delete(deletePerson)
```

Comment:
I'm not a fan of reusing the `route` calls for multiple operations. Even though they're the same now, they don't serve the same purpose and something might change in one of them that makes you separate. In this case, I also don't think the duplication is enough to sacrifice the readability. Take a look at this version, it's much easier to spot all the operations in for this router:

```suggestion
router.get("/", getPeople);
router.post("/", addPerson);
router.get("/:id", getPerson);
router.put("/:id", updatePerson);
router.delete("/:id", deletePerson);
```